mpt_api_client/http/async_client.py

```diff
@@ -49,33 +54,55 @@ def __init__(
             "Authorization": f"Bearer {api_token}",
             "Accept": "application/json",
         }
-        super().__init__(
+        self.httpx_client = AsyncClient(
             base_url=base_url,
             headers=base_headers,
             timeout=timeout,
             transport=AsyncHTTPTransport(retries=retries),
         )
 
-    @override
     async def request(  # noqa: WPS211
         self,
         method: str,
-        url: URL | str,
+        url: str,
         *,
-        content: RequestContent | None = None,  # noqa: WPS110
+        content: ContentType | None = None,  # noqa: WPS110
         data: RequestData | None = None,  # noqa: WPS110
         files: RequestFiles | None = None,
         json: Any | None = None,
-        params: QueryParamTypes | None = None,  # noqa: WPS110
+        params: QueryParam | None = None,  # noqa: WPS110
```

Comment:
Why not call it query_params to remove the noqa? Same comment for content and data